mlflow/utils/databricks_utils.py

```diff
@@ -419,6 +430,31 @@ def _fail_malformed_databricks_auth(profile):
     )
 
 
+def _fail_model_serving_creds_env(exception):
+    raise MlflowException(
+        "Unable to read Oauth credentials from file mount for Databricks "
+        f"Model Serving dependency, failed with exception: {exception}"
+    )
```

Comment:
```suggestion
    raise MlflowException(
        "Unable to read Oauth credentials from file mount for Databricks "
        f"Model Serving dependency failed"
    ) from exception
```

I think this is the more pythonic way to do it